code/modules/mob/living/simple_animal/bot/medbot.dm

```diff
@@ -92,15 +95,11 @@
 
 /mob/living/simple_animal/bot/medbot/Initialize()
 	..()
-	update_icon()
-
-	if(skin)
-		add_overlay("medskin_[skin]")
-
 	var/datum/job/doctor/J = new /datum/job/doctor
 	access_card.access += J.get_access()
 	prev_access = access_card.access
 	qdel(J)
+	addtimer(CALLBACK(src, .proc/update_icon), 1)

```

Comment:
Hmm, that should fix it. I'm on mobile but I'll do that once I return to Casa de Xhuis.